cfgov/unprocessed/css/pages/ask.less

```diff
@@ -30,4 +30,45 @@
     ol:last-child {
         margin-bottom: 0 !important;
     } 
+}
+
+.ask-landing-page {
+    .content_main {
+        padding-top: 0;
+        padding-bottom: 0;
+        
+        .respond-to-range(@bp-sm-min, @bp-med-max, {
+            border-width: 0;
+        });
+    }
+
+    .question-categories {
+        .respond-to-min(@bp-sm-min, {
```

Comment:
I looked at using block__bg, but since the background color isn't applied on mobile sizes, it still require some page-specific adjustments. This approach is not ideal, though, so I'll think about it some more.